tests/create_buckets.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+from boto3 import Session
+from kubernetes import client, config
+import time
+import sys
+import os
+
+
+def get_env(key, default=None, error=False):
+    if not error:
+        return os.environ.get(key, default)
+    return os.environ[key]
+
+
+def bucket_safe_create(bucket):
```

Comment:
IIRC, this function is defined here already: https://github.com/scality/Zenko/blob/5a3318738a22752b1453fab0e38e8458d52883f1/tests/zenko_e2e/util/bucket.py#L11

My python is rusty, is there a reason why this is being redefined instead of being imported?

Reply:
create_buckets is just a one off script, while zenko_e2e is a separate project. I'd prefer no to entangle the two over saving a few lines.